### PR TITLE
use ENV for client specific logic

### PR DESCRIPTION
**Pull request by leonardow-unep-wcmc**
Once this PR merged to IAC branch, we can merge IAC branch back to master.

### CODE REVIEW

File: config/deploy/iacseaturtle-ort/staging.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 set :stage, :staging
 
-set :branch, 'iacseaturtle-master'
+set :branch, 'master'
```
Comment (lucacug) [inline config/deploy/iacseaturtle-ort/staging.rb:3]: usually we deploy `develop` to staging as per default, but I don't know how many differences there will be between master and develop

File: app/models/section.rb [hunk 2]
```diff
@@ -265,7 +265,48 @@ def update_submission_state_to_care user, looping_identifier=nil
...
+    if ENV['ORS_CLIENT_CODE'] == 'IAC'
+      initialise_tree_submission_states_iac_version(user, loop_sources_items, loop_item=nil, dont_care_descendants=false, looping_identifier=nil)
+    else
+      initialise_tree_submission_states_default_version(user, loop_sources_items, loop_item=nil, dont_care_descendants=false, looping_identifier=nil)
+    end
+  end
+
+  def initialise_tree_submission_states_default_version(user, loop_sources_items, loop_item=nil, dont_care_descendants=false, looping_identifier=nil)
+    state_tracker = UserSectionSubmissionState.find_or_initialize_by_section_id_and_user_id_and_looping_identifier(self.id, user.id, looping_identifier)
+    #dont_care_descendants = dont_care_descendants || (self.depends_on_question.present? && !self.dependency_condition_met?(user, loop_item))
```
Comment (lucacug) [inline app/models/section.rb:278]: can this comment be removed?

**Review by lucacug — COMMENTED** [review]
just a couple of minor comments

File: config/deploy/iacseaturtle-ort/staging.rb (hunk 1 above)
Comment (leonardow-unep-wcmc) [reply config/deploy/iacseaturtle-ort/staging.rb:3]: Other MEA's staging use `master` as well for staging. Not ideal but just follow others for now.

File: app/models/section.rb (hunk 2 above)
Comment (leonardow-unep-wcmc) [reply app/models/section.rb:278]: The comment is from original code. Just try to keep it as original as before.

**Pull request merged**